funnel/views/email.py

```diff
@@ -53,11 +58,18 @@ def send_email(subject, to, content):
         for recipient in to
     ]
     body = html2text(content)
-    html = transform(content, base_url=request.url_root)
+    html = transform(
+        content, base_url=f'https://{app.config["DEFAULT_DOMAIN"]}/')  # NOQA
```

Comment:
Never use naked `# NOQA`. Specify the code you're ignoring. Also, why does this line have a QA problem?